content/pt-br/docs/tasks/administer-cluster/limit-storage-consumption.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Limitar o consumo de armazenamento
+content_type: task
+weight: 240
+---
+
+<!-- overview -->
+
+Este exemplo demonstra como limitar a quantidade de armazenamento consumido em um namespace.
+
+Os seguintes recursos são usados na demonstração: [`ResourceQuota`](/pt-br/docs/concepts/policy/resource-quotas/), [`LimitRange`](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/), e [`PersistentVolumeClaim`](/pt-br/docs/concepts/storage/persistent-volumes/).
```

Comment:
```suggestion
Os seguintes recursos são usados na demonstração: [ResourceQuota](/pt-br/docs/concepts/policy/resource-quotas/), [LimitRange](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/), e [PersistentVolumeClaim](/pt-br/docs/concepts/storage/persistent-volumes/).
```

Following the [Kubernetes Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), it's preferable to not stylize API object names as code.